### PR TITLE
Add tests for CertifOps: encrypt_for_realm

### LINKED ISSUE #6326: encrypt_for_realm

### PR DESCRIPTION
**Pull request by TimeEngineer**
closes #6326

### CODE REVIEW

File: libparsec/crates/client/tests/unit/certif/encrypt_for_realm.rs [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+        },
+    );
+
+    let err = ops.encrypt_for_realm(realm_id, b"data").await.unwrap_err();
+
+    assert(err);
+}
+
+#[parsec_test(testbed = "minimal")]
+async fn empty(env: &TestbedEnv) {
```
Comment (FirelightFlagboy) [inline libparsec/crates/client/tests/unit/certif/encrypt_for_realm.rs:109]:
```suggestion
async fn unknown_realm(env: &TestbedEnv) {
```

File: libparsec/crates/client/tests/unit/certif/encrypt_for_realm.rs [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+    let (env, realm_id) = env.customize_with_map(|builder| {
+        builder
+            .new_realm("alice")
+            .then_do_initial_key_rotation()
+            .map(|event| event.realm)
+    });
+    let alice = env.local_device("alice@dev1");
+    let ops = certificates_ops_factory(&env, &alice).await;
+
+    ops.add_certificates_batch(
```
Comment (touilleMan) [inline libparsec/crates/client/tests/unit/certif/encrypt_for_realm.rs:23]: use `builder.certificates_storage_fetch_certificates("alice@dev1");` instead

File: libparsec/crates/client/tests/unit/certif/encrypt_for_realm.rs [hunk 3]
```diff
@@ -0,0 +1,119 @@
...
```
Comment (touilleMan) [inline libparsec/crates/client/tests/unit/certif/encrypt_for_realm.rs:84]: use `builder.certificates_storage_fetch_certificates("alice@dev1");`

File: libparsec/crates/client/tests/unit/certif/encrypt_for_realm.rs (hunk 2 above)
Comment (FirelightFlagboy) [reply libparsec/crates/client/tests/unit/certif/encrypt_for_realm.rs:23]: You can suggest code directly in the comment, see [Adding comments to a PR](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/commenting-on-a-pull-request#adding-comments-to-a-pull-request) at the step `6`

**Pull request merged**